lib/querystring.js

```diff
@@ -230,7 +229,7 @@ QueryString.parse = QueryString.decode = function(qs, sep, eq, options) {
 
   var keys = [];
   for (var i = 0; i < len; ++i) {
-    const x = qs[i].replace(regexp, '%20');
+    const x = replacePlus(qs[i]);
```

Comment:
There's very little difference in performance. v8 is pretty good at regexps, especially simple ones like `/\+/`.